### ISSUE TITLE
Mailing list browser down

### ISSUE THREAD

**Issue by jmagnusson** [benoitc/gunicorn#1362]
http://lists.gunicorn.org/user

Google says:

"This site can’t be reached
lists.gunicorn.org’s server DNS address could not be found."

**Comment by berkerpeksag** [comment]
Thanks for the report. This is a known problem and we are working on it. See [this](https://twitter.com/benoitc/status/780459153819111424) tweet for details.

**Comment by benoitc** [comment]
working on it. It won't be on python.org servers though due to their CoC. I will setup that this week.

**Comment by tilgovi** [comment]
@benoitc what about the CoC prevents us from using python.org?

**Comment by benoitc** [comment]
@tilgovi this si only that it has a CoC. 

On the other hand I still have to setup this mailing list. I will try sometimes this week..

**Comment by tilgovi** [comment]
Is this the CoC? https://www.python.org/psf/codeofconduct/

If we adopt it can we use python.org infrastructure?

I do not have any problem with this text, myself.

**Comment by benoitc** [comment]
I decided to not use a mailing list and only use [Github](https://github.com/benoitc/gunicorn/projects) for the project manaement and discussions. Some projects Like spacemacs are already doing it with success.

I created a wiki that contains the rules: https://github.com/benoitc/gunicorn/wiki/Project-management 

will update the website later today and reference github issues to each projects today so it will be annonced with the release tomorrow.
